Tasks/QuickPerfTest/Invoke-QuickPerfTest.ps1

```diff
@@ -89,11 +91,19 @@ function UploadTestDrop($testdrop)
 function GetTestRuns($headers)
 {
     $uri = [String]::Format("{0}/_apis/clt/testruns?api-version=1.0", $CltAccountUrl)
-    $runs = Invoke-RestMethod -ContentType "application/json" -UserAgent $userAgent -Uri $uri -Headers $headers
+    $runs = Invoke-RestMethod -ContentType "application/json" -UserAgent $userAgent -TimeoutSec $global:RestTimeout -Uri $uri -Headers $headers
 
     return $runs
 }
 
+function GetTestRunUri($testRunId)
```

Comment:
>GetTestRunUri [](start = 9, length = 13)

Vinit, all REST API helper methods actually take the headers as input. please see if you can stick to the same pattern.